l.js

```diff
@@ -29,6 +29,7 @@
 		, readyState = 'readyState'
 		, onreadystatechange = 'onreadystatechange'
 		, typeJS = 'text/javascript'
+		, typeModule = 'module'
```

Comment:
no need to create a var for this as it is used only once in the rest of the code. The purpose of such declaration is to obtain better minification so without reusing the string multiple time we can simply use the original string. With the modification proposed, we can also remove the typeJs var as we won't write it multiple time either.